### PR TITLE
Update 02-Lightning-Lab-1.md

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
In 1st question of this lab. The version which needs to be update is 1.26 not 1.24. I this solutions are old and needs an update. I will try to change if I get something in future. Thanks

### CODE REVIEW

**Comment by Sajiyah-Salat** [comment]
Ohk Great then I will make changes. Thank you

**Comment by Sajiyah-Salat** [comment]
I have made the changes. Kindly review.

File: docs/14-Lightning-Labs/02-Lightning-Lab-1.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
         ```
         apt-mark unhold kubelet
-        apt-get install -y kubelet=1.24.0-00
+        apt-get install -y kubelet=1.25.0-00
```
Comment (fireflycons) [inline docs/14-Lightning-Labs/02-Lightning-Lab-1.md:59]: This should be `1.26.0-00`

File: docs/14-Lightning-Labs/02-Lightning-Lab-1.md [hunk 2]
```diff
@@ -71,7 +71,7 @@
 
         ```
         apt-mark unhold kubectl
-        apt-get install -y kubectl=1.24.0-00
+        apt-get install -y kubectl=1.25.0-00
```
Comment (fireflycons) [inline docs/14-Lightning-Labs/02-Lightning-Lab-1.md:74]: This should also be `1.26.0-00`

**Comment by fireflycons** [comment]
Since the labs themselves have been upgraded to 1.26, there may be a fair amount of other pages in this repo that need updating. Please feel free to keep sending me PRs. Correcting this material is a good way to revise for the exam 😄

**Comment by Sajiyah-Salat** [comment]
Great, I would love to do that. Help me to get exact location of files. Let me know if any further changes needed. Thank you.

**Comment by fireflycons** [comment]
Looks good now.
I have a few pages I'm editing in sections 9 and 11, and I'm also going to rewrite the kubeadm install on virtualbox.
Anything else you see, please feel free.

**Comment by fireflycons** [comment]
I see you have a fork of this repo - Keep it up to date and you can submit PRs directly from your fork instead of inline edits.

**Comment by Sajiyah-Salat** [comment]
Sure, I will try to contribute here for sure. Thank you.

**Pull request merged**